src/DynamoWPFCLI/CommandLineRunner.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml;
+using System.Threading;
+using System.IO;
+using DesignScript.Builtin;
+using Dynamo.Models;
+using Dynamo.Applications;
+using Dynamo.Graph.Workspaces;
+using Dynamo.ViewModels;
+using Dynamo.Visualization;
+using Newtonsoft.Json;
+
+namespace DynamoWPFCLI
+{
+    /// <summary>
+    /// This class invokes a dynamo model's run methods in a headless mode from the CLI using a set of flags.
+    /// This class also has a very limited method for exporting the graph evaluation to an xml file, so that 
+    /// the results from invoking dynamo from the command line are useable.
+    /// </summary>
+    public class CommandLineRunner
```

Comment:
doesn't it make sense for this to extend CommandLineRunner and override its methods - seems weird to have two classes with the same name in the solution.